src/main/java/nl/novi/techiteasycontroller/Model/Television.java

```diff
@@ -0,0 +1,4 @@
+package nl.novi.techiteasycontroller.Model;
+
+public class Television {
```

Comment:
Deze mag je nog aanvullen met attributen, getter/setters en eventueel consturctor(s)

Reply:
Ok check, added to java class.